connector_importer/components/importer_csv_std.py

```diff
@@ -0,0 +1,165 @@
+                continue
+            # Collect tracker data for later
+            # We store the parameters for chunk_report.track_{created,updated}
+            # functions, excepted the odoo_record which could not be known
+            # for newly created records
+            odoo_record_exists = self.record_handler.odoo_exists(
+                values, line, use_xmlid=True)
+            if odoo_record_exists:
+                odoo_record = self.record_handler.odoo_find(
+                    values, line, use_xmlid=True)
+                tracker_data['updated'][i] = [line, odoo_record]
+            else:
+                tracker_data['created'][i] = [line, None]
+
+            # handle forced skipping
+            skip_info = self.skip_it(values, line)
+            if skip_info:
+                self.tracker.log_skipped(values, line, skip_info)
+                continue
+            mapped_lines.append(values)
+
+        if mapped_lines:
+            try:
+                with self.env.cr.savepoint():
+                    fieldnames, data = self.prepare_load_params(mapped_lines)
+                    load_res = self.model.load(fieldnames, data)
+
+                    # Log load errors
+                    for message in load_res['messages']:
+                        if message.get('rows'):
+                            line_numbers = range(
+                                message['rows']['from'],
+                                message['rows']['to'] + 1)
+                            for line_nr in line_numbers:
+                                # First we remove the entry from tracker data
+                                tracker_data['created'].pop(line_nr, None)
+                                tracker_data['updated'].pop(line_nr, None)
+                                # We add 2 as the tracker count lines starting
+                                # from 1 + header line
+                                line = {'_line_nr': line_nr + 2}
+                                self.tracker.log_error(
+                                    {}, line, odoo_record=None,
+                                    message=message['message'])
+                        else:
+                            line = {'_line_nr': 0}
+                            self.tracker.log_error(
+                                {}, line, odoo_record=None,
+                                message=message['message'])
+            except Exception as err:
+                line = {'_line_nr': 0}
+                self.tracker.log_error(
+                    {}, line, odoo_record=None, message=err)
+                if self._break_on_error:
+                    raise
+
+        # Record the tracker data for the report
+        # We are using 'tracker.chunk_report' methods instead of 'tracker'
+        # as the formers don't require an 'odoo_record' (which we don't have)
+        for args in tracker_data['created'].values():
+            self.tracker.chunk_report.track_created(
```

Comment:
we can modify `log_*` methods on the tracker to make `odoo_record` optional. It will still work by position for existing implementations.

Reply:
I see the modification in the signature of the method but these lines are the same... :thinking: